managed-ledger/src/main/proto/MLDataFormats.proto

```diff
@@ -78,6 +81,11 @@ message MessageRange {
     required NestedPositionInfo upperEndpoint = 2;
 }
 
+message BatchDeletedIndexInfo {
+    required NestedPositionInfo position = 1;
+    repeated int64 ackBitSet = 2;
```

Comment:
A few comments here:

1) I would prefer calling it `delete_set`. "Acknowledgement" is the concept at the client & broker side. In managed ledger level, we use "delete". So it is better to keep the name consistent.

2) I am not sure we need to use `repeated` here.